android/app/build.gradle

```diff
@@ -48,7 +48,7 @@ android {
         applicationId "org.micromentor.mm_flutter_app"
         // You can update the following values to match your application needs.
         // For more information, see: https://docs.flutter.dev/deployment/android#reviewing-the-gradle-build-configuration.
-        minSdkVersion flutter.minSdkVersion
+        minSdkVersion 19
```

Comment:
Some of the new dependencies require a min sdk of 19. The flutter min is 16 by default, which is [extremely old](https://apilevels.com/). In fact Im bumping it up to version 21 since less than 1% of devices are using SDKs that low anyway. Also I believe we need to support newer versions to publish to Google Play.